.dockerignore

```diff
@@ -0,0 +1,8 @@
+.dockerignore
+*.md
+node_modules
+build
+client
+.github
+dist
+.env
```

Comment:
I believe this file should be in the [`server` folder](https://github.com/KanaryStack/aliascheck/tree/68233d431d50356bb8c4d83134b99f99f0f6536f/server).

Reply:
The `.dockerignore` location is guided by the location of the dockerfiles. See https://github.com/KanaryStack/aliascheck/pull/26#discussion_r952086936

Reply:
Please refer to [my earlier comment addressing the preferred location of the `Dockerfile`](https://github.com/KanaryStack/aliascheck/pull/26#discussion_r952487845).